experimental/dds/tree/src/test/IdCompressor.perf.tests.ts

```diff
@@ -1,29 +1,16 @@
+/* eslint-disable @typescript-eslint/no-non-null-assertion */
```

Comment:
will give you instant approve if you sneak this into the .eslintrc.js file for this package

Reply:
no u